chain/db/table.go

```diff
@@ -2,18 +2,35 @@ package db
 
 import (
 	"encoding/json"
-
-	"github.com/jinzhu/gorm"
 )
 
 type Metadata struct {
 	Key   string `gorm:"primary_key"`
 	Value string `gorm:"not null"`
 }
 
-type Event struct {
-	gorm.Model
-	Name string
+type Block struct {
+	Height    int64  `gorm:"primary_key;auto_increment:false"`
+	Timestamp int64  `gorm:"not null"`
+	Proposer  string `gorm:"not null"`
+	BlockHash []byte `gorm:"not null"`
+}
+
+type Transaction struct {
+	TxHash      []byte          `gorm:"primary_key"`
+	Timestamp   int64           `gorm:"not null"`
+	GasUsed     int64           `gorm:"not null"`
+	GasLimit    uint64          `gorm:"not null"`
+	GasFee      string          `gorm:"not null"`
+	Sender      string          `gorm:"not null"`
+	Success     bool            `gorm:"not null"`
+	BlockHeight int64           `gorm:"not null"`
+	Messages    json.RawMessage `sql:"json;not null"`
+}
+
+type Account struct {
+	Address string `gorm:"primary_key"`
+	Balance string `gorm:"not null"`
```

Comment:
Let's store UpdatedHeight here. I think it will help us debug a lot in the future.